diff --git a/lib/parser.js b/lib/parser.js
index 3e83fdc9f6cb975b025aa6875723f7cd5652376e..a1a74a1e47b770df6f31f93fdb087cc37f9b0e9d 100644
--- a/lib/parser.js
+++ b/lib/parser.js
@@ -19,7 +19,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.getDefaultExportForFile = exports.Parser = exports.withCompilerOptions = exports.withCustomConfig = exports.withDefaultConfig = exports.parse = exports.defaultOptions = exports.defaultParserOpts = void 0;
 var fs = require("fs");
 var path = require("path");
-var ts = require("typescript");
+var ts = require("@typescript/typescript6");
 var buildFilter_1 = require("./buildFilter");
 var trimFileName_1 = require("./trimFileName");
 exports.defaultParserOpts = {};
